### PR TITLE
[Intel MKL] Enabling more operator fusions

### PR DESCRIPTION
**Pull request by nhasabni**
Support for Conv+ReLU fusion; Enabling Shape op as a part of MKL layout propagation

1) This commit adds support for Conv+ReLU and BatchNorm+ReLU fusion and corresponding graph unit
tests. 2) This PR also adds support for Shape op in MKL layout propagation. 3) It also updates
MklToTf conversion pass unit tests because of that change.

### CODE REVIEW

**Comment by tensorflowbutler** [comment]
Nagging Assignee @benoitsteiner: It has been 14 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by drpngx** [comment]
So sorry this fell through the cracks.

@tatianashp could you take a look?

**Comment by tensorflowbutler** [comment]
Nagging Reviewer @tatianashp: It has been 15 days with no activity and the `awaiting review` label was assigned. Can you please take a look?

**Comment by tensorflowbutler** [comment]
Nagging Reviewer @tatianashp: It has been 36 days with no activity and the `awaiting review` label was assigned. Can you please take a look?

**Comment by tatianashp** [comment]
@nhasabni Sorry it fell through the cracks again. Very large PRs are hard to review. Could you please resolve conflicts and split the changes into a few smaller PRs. I'll review then.

**Comment by tensorflowbutler** [comment]
Nagging Assignee @benoitsteiner: It has been 14 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by tensorflowbutler** [comment]
Nagging Assignee @benoitsteiner: It has been 29 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by tensorflowbutler** [comment]
Nagging Assignee @benoitsteiner: It has been 44 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by tatianashp** [comment]
There has been no activity on this PR for a while.
@nhasabni Could you please either resolve conflicts or close the PR?
/cc @agramesh1

**Comment by tensorflowbutler** [comment]
Nagging Assignee @benoitsteiner: It has been 14 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by tatianashp** [comment]
Closing due to lack of recent activity. If this PR is still relevant please resolve conflicts and re-open.
